lib/wallet/wallet.js

```diff
@@ -82,10 +82,17 @@ class Wallet extends EventEmitter {
 
     const bidsCacheSizeInBlocks = 20;
     const bidsPerAuction = 3; // average in March 2020 is ~3.1
-    const capacity =
+
+    // TODO (bennett) - make this configurable from options
+    const bidsCapacity =
       consensus.MAX_BLOCK_OPENS * bidsPerAuction * bidsCacheSizeInBlocks;
-    this.sendBidResults = new LRU(capacity);
+    this.sendBidResults = new LRU(bidsCapacity);
+
+    // TODO (bennett) - make this configurable from options
+    const opensCapacity = consensus.MAX_BLOCK_OPENS * opensCacheSizeInBlocks;
```

Comment:
Variable not defined `opensCacheSizeInBlocks`

Reply:
Whoops, deleted wrong line